src/apps/csvToJson.jsx

```diff
@@ -11,8 +11,6 @@ import Breakpoints from "../shared/Breakpoints";
 const Container = styled.div`
   display: flex;
   flex-direction: column;
-  justify-content: center;
```

Comment:
Removing this would make it look bad for smaller devices. 
![image](https://user-images.githubusercontent.com/60931421/133929659-9b2fb892-d938-4ed3-8a7b-55b14becb804.png)

Reply:
oh, we need a media query here then. I'll create a ticket for improving `csvToJson` app.